atlp-rwanda/trojans-ec-fe
# #000000 - Add logout

## What does This PR do 

this PR is for any user to logout the website 

## Description of the Task completed ?

given that you are already logged in as either a **buyer, seller, or an admin** and after you have visited our website you want to logout the app you can use this functionality to logout 

## How can it be manually tested 

to test this feature you can log in on our website and then click on the dashboard icon and you will be directed to the dashboard and upon reaching there you will see a sidebar and hover on that sidebar and at the botton of the LI you will see the logout button 

here is the [PR Preview](https://trojans-ec-fe-git-ft-add-logout-000000-atlp-trojan.vercel.app/)

## any background context you want to provide ?

you need to log in to test this feature

## what are the relevant pivotal tracker stories?

N/A

## screenShots(if appropriate)

### the logout button

<img width="719" alt="Screenshot 2023-05-09 at 3 45 46 PM" src="https://github.com/atlp-rwanda/trojans-ec-fe/assets/111174174/46480577-8b12-477b-a0b0-048679d02eaa">


### when you try to access the Homepage after you've logged out 

<img width="719" alt="Screenshot 2023-05-09 at 3 48 56 PM" src="https://github.com/atlp-rwanda/trojans-ec-fe/assets/111174174/50c883af-b1d5-413e-800f-b63c13ddf8d7">
